test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -27,6 +27,9 @@
  */
 
 #include "connection_simulator.h"
+#include "timestamp_manager.h"
+#include <iostream>
```

Comment:
Do we need `iostream` here?

Reply:
It seems so. Maybe I'm including things in the wrong order?

Reply:
[here](https://github.com/wiredtiger/wiredtiger/pull/8116/files#r920051707)